riotfile.py

```diff
@@ -202,7 +205,7 @@ def select_pys(min_version=MIN_PYTHON_VERSION, max_version=MAX_PYTHON_VERSION):
         Venv(
             name="integration",
             # Enabling coverage for integration tests breaks certain tests in CI
-            command="pytest --no-cov {cmdargs} tests/integration/",
+            command="pytest --no-cov --ddtrace --ignore-glob='*civisibility*' {cmdargs} tests/integration/ && pytest --no-cov {cmdargs} tests/integration/test_integration_civisibility.py",  # noqa: E501
```

Comment:
comment here would be nice, why do we need to break them up? I assume we cannot run with `--ddtrace` for civisibility integration tests?

Reply:
Yes, that's the reason, I can add a comment